controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -2438,7 +2438,7 @@ public interface ControllerLogger extends BasicLogger {
      *
      * @return an {@link IllegalStateException} for the error.
      */
-    @Message(id = 214, value = "Could not get main file: %s. Specified files must be relative to the configuration dir: %s")
+    @Message(id = 214, value = "Could not get main file: %s. the configuration dir: %s could be set by the property jboss.server.config.dir during jboss-cli start.")
```

Comment:
Capitalize the first character of the sentence? `the configuration ... `
Is this really limited to `jboss-cli` only? I thought it applies for server start in general.

Anyway, IMHO I would still find this misleading, maybe even more than the previous message which at least tries to tell what is wrong. 

I think the main focus of the message needs to be that `only files directly in configuration dir can be loaded` (I hope I understood the changed implementation correctly).
Of course you can also add the additional info how to change the configuration dir, as an enhancement to the main message.

Reply:
Thanks! @jbliznak for the review. I agree with your point but the thing is users already getting the idea as the relative path which was provided by `Could not get main file: %s.` If we can provide a solution to them, that will be a good idea.
maybe we can use this generalized logger message instead:
"Could not get main file: %s. Files directly in configuration dir: %s can be loaded which could be set by the property jboss.server.config.dir"

Reply:
> users already getting the idea as the relative path which was provided by Could not get main file: %s.

I don't think the users are getting the idea what is wrong:
```
./standalone.sh --server-config=../../docs/examples/configs/standalone-jts.xml

WFLYCTL0214: Could not get main file: ../../docs/examples/configs/standalone-jts.xml. Specified files must be relative to the configuration dir: /tmp/wildfly-26.1.1.Final/standalone/configuration
```
Given the original message I am lost because the `../../docs/examples/configs/standalone-jts.xml` is a valid relative path to the configuration dir pointing to existing configuration so everything seems to be ok. (that was the reason for the jira I guess)
The whole point is that the implementation changed and doesn't allow files outside of configuration dir to be loaded anymore so the message stopped to be correct (it was correct before). So we must tell a user that this is the case.

My proposals would be one of these:
```
// adding the keyword `only` so that the restriction is obvious
@Message(id = 214, value = "Could not get main file: %s. Only files from the configuration dir %s could be loaded.")

// adding also possible solution as you wanted
@Message(id = 214, value = "Could not get main file: %s. Only files from the configuration dir %s could be loaded. Configuration dir location can be changed by setting the property jboss.server.config.dir.")
```

BTW isn't there some constant for `jboss.server.config.dir` to make it safe? I don't know if that is required though.

Reply:
Thanks, Got your point and I'm going with your 1st suggestion.
About the constant for `jboss.server.config.dir` I haven't looked into that much as it was not mentioned in Jira. will take a look.